content/css/concepts/floats/floats.md

```diff
@@ -48,7 +48,7 @@ The `float` property primarily uses the following three values:
   <br />
   <br />
   <br />
-  <p>This example shows CSS floats and I wont float at all.</p>
+  <p>This example shows CSS floats and I will not float at all.</p>
```

Comment:
Since we're changing the text, can we create a new image and remove the old one (line 79)

Reply:
@garanews once we resolve this comment, I think this PR should be merge-ready!